modules/nextflow/src/main/groovy/nextflow/processor/PublishDir.groovy

```diff
@@ -504,7 +504,14 @@ class PublishDir {
     }
 
     protected void createPublishDir() {
-        makeDirs(this.path)
+        try {
+            makeDirs(path)
+        }
+        catch( Throwable e ) {
+            log.warn "Failed to create publish directory: ${path.toUriString()} -- See log file for details", e
+            if( NF.strictMode || failOnError )
+                session?.abort(e)
+        }
```

Comment:
If it fails to create the path does it make sense it continue the publishing the operation?

Think it would be better capture and re-trow e.g. 

```
  throw new IllegalStateException("Failed to create publish directory: ${path.toUriString()}", e)
```

Reply:
Good point

Reply:
```suggestion
        try {
            makeDirs(path)
        }
        catch( Throwable e ) {
            session?.abort(new IllegalStateException("Failed to create publish directory: ${path.toUriString()}", e))
        }
```